src/docs/development/platform-integration/web.md

```diff
@@ -9,32 +9,36 @@ As of the 1.12 release, web support for Flutter
 is available on the beta channel.
 **We don't recommend deploying a web app to production.**
 
+### What sort of apps are not ideal for Flutter on the web?
+
```

Comment:
To answer the question of what scenarios are ideal, we can either summarize what we have listed on flutter.dev/web here or just link them to the page and make the scenarios more explicit on that page.

"While traditional web approaches remain a great choice, we envision the web support available in Flutter being valuable in many scenarios. For example:

Rich interactive single page application
Connected Progressive Web Application 
Embedding dynamic/interactive content in an existing website 


^^ these need to be updated in flutter.dev/web... ill submit a PR soon for updates there.